controller/comments.controller.js

```diff
@@ -0,0 +1,10 @@
+const { fetchCommentsById } = require("../model/comments.model");
+
+exports.getCommentsById = (req, res, next) => {
+  const { article_id } = req.params;
+  fetchCommentsById(article_id)
+    .then((result) => {
+      res.status(200).send({ result });
```

Comment:
One small change `result` isnt very descriptive as to what the data is. 
Could you change this to something that is, please?